dhall/doctest/Main.hs

```diff
@@ -24,10 +26,5 @@ main = do
         writeFile "file2" "./file1"
         writeFile "file1" "./file2"
 
-        Test.DocTest.doctest
-            [ "--fast"
-            , "-i" <> (prefix </> "src")
-            , prefix </> "src/Dhall.hs"
-            , prefix </> "src/Dhall/Import.hs"
-            , prefix </> "src/Dhall/Tutorial.hs"
-            ]
```

Comment:
To be honest I don't know what purpose the command line's serving here. If any of this is pulling weight - maybe `--fast`? - it should go back in, but things seem to work without.

Reply:
I think it's mainly because it was written without integrating with Cabal (i.e. running `doctest` as it would have been run naively from the command line)

Reply:
I think I've figured this out in https://github.com/dhall-lang/dhall-haskell/pull/1433.